_posts/2022-11-29-no83-hacks-sink-in.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: post
+title: Berlin Hack & Tell \#83 - Hacks sink in
+date: 2022-09-29
+time: '19:00'
+location: '[c-base](https://www.c-base.org)'
+meetupUrl: https://www.meetup.com/de-DE/berlin-hack-and-tell/events/289908789/
+---
+
+* [proofinprogress](https://github.com/timdaub/proofinprogress.com) by [Tim](https://github.com/timdaub) - Subtract, don't add and overcomplicate your website!
+* Gateway a board game by AFK - two parallel worlds, bring the six creatures back without violence- **Hack of the Month**
+* AI-generated [conversation card game](https://cards.langa.me) by [Langame](https://github.com/langa-me)
+* [ez-ssh-bot](https://github.com/echtzeit-dev/ez-ssh-bot) by Stefan - reverse SSH Telegram bot
+* [Cyberfame](https://cyberfame.io) by 0xAverageUser - security, manuals, UA, git repos, CVE scanning, remediation, OSS
+* FlashCards by Sam. [Application logic visualisation](https://stately.ai) demonstration on a FlashCards app
+* [Gemini protocol](https://gemini.circumlunar.space/docs/) by carl
```

Comment:
```suggestion
* [WikDict on Gemini protocol](https://github.com/karlb/wikdict-gemini) by [Karl](https://www.karl.berlin/)
```

Reply:
thanks and sorry for butchering your name - but this one I just copied from the pad where people can put notes.

Reply:
No need to be sorry. As you noted, it's my fault for not supplying the information earlier. Thanks for organizing BHNT!